packages/page-accounts/src/modals/ProxyOverview.tsx

```diff
@@ -46,23 +46,23 @@ const optTxBatch = { isBatchAll: true };
 
 const EMPTY_EXISTING: [PalletProxyProxyDefinition[], BN] = [[], BN_ZERO];
 
-function createAddProxy (api: ApiPromise, account: AccountId, type: NodeRuntimeProxyType, delay = 0): SubmittableExtrinsic<'promise'> {
+function createAddProxy(api: ApiPromise, account: AccountId, type: NodeRuntimeProxyType, delay = 0): SubmittableExtrinsic<'promise'> {
```

Comment:
```suggestion
function createAddProxy (api: ApiPromise, account: AccountId, type: NodeRuntimeProxyType, delay = 0): SubmittableExtrinsic<'promise'> {
```

Fix, CI fails linting due to this.